src/Scaffolding/VS.Web.CG.Mvc/Templates/Blazor/Delete.cs

```diff
@@ -54,9 +54,9 @@ public virtual string TransformText()
 
             this.Write("@inject ");
             this.Write(this.ToStringHelper.ToStringWithCulture(dbContextFactory));
-            this.Write("\r\n@inject NavigationManager NavigationManager\r\n\r\n\r\n<PageTitle>Delete</PageTitle>\r" +
-                    "\n\r\n<h1>Delete</h1>\r\n\r\n<h3>Are you sure you want to delete this?</h3>\r\n<div>\r\n   " +
-                    " <h4>");
+            this.Write("\r\n@inject NavigationManager NavigationManager\r\n\r\n<PageTitle>Delete</PageTitle>\r\n\r" +
+                    "\n<h1>Delete</h1>\r\n\r\n<h3>Are you sure you want to delete this?</h3>\r\n<div>\r\n    <" +
```

Comment:
I just noticed this ... `Delete`, `Edit`, `Create`, and `Details` components skip heading levels (e.g., `<h1>` to `<h3>`/`<h4>`), which isn't an SEO best practice. Seems like a paragraph would be best for a line of text (`<p>`) (the delete confirmation question); but either way on that, I recommend at least numbering the heading elements sequentially in these four components.

Reply:
should I change to h1, h2, and h3 for now instead of the currently h1, h3 and h4? @guardrex

Reply:
also for these changes, I would prefer if you comment on the .tt files instead of the autogenerated .cs files

Reply:
I yield to @danroth27 on if the confirmation question should be in a paragraph (`<p>`).

Otherwise, I think we're all in agreement the headings should be sequentially numbered. 

It's either of .....

* `<h1>` ... `<p>` ... `<h2>`
* `<h1>` ... `<h2>` ... `<h3>`

Reply:
> prefer if you comment on the .tt files

GH won't allow commenting on those lines on the diff.

Reply:
ah gotcha, thats annoying for y'all, its fine for now, i was just being picky lol

Reply:
Let's go with @guardrex's suggestion making the heading sequential and turning the "Are you sure you want to delete this?" text into a `p` tag.

Reply:
sounds good tyty